README.md

```diff
@@ -48,4 +48,4 @@ yarn generate
 
 To preview the static generated app, run `yarn start`
 
-For detailed explanation on how things work, checkout [nuxt/content](https://content.nuxtjs.org) and [@nuxt/content theme docs](https://content.nuxtjs.org/themes-docs).
+For detailed explanation on how things work, checkout [nuxt/content](https://github.com/nuxt/content).
```

Comment:
I would keep both the frontend site https://content.nuxtjs.org and add the GitHub here

Reply:
So, something like this? 👇🏼 

For detailed explanation on how things work, checkout [nuxt/content](https://content.nuxtjs.org) & it's [github repository](https://github.com/nuxt/content).

Reply:
```suggestion
For detailed explanation on how things work, checkout [nuxt/content](https://github.com/nuxt/content) and their official [GitHub repo](https://github.com/nuxt/content).
```